sdks/python/scripts/run_integration_test.sh

```diff
@@ -48,6 +48,8 @@
 #                      during execution. Commonly used options like `--attr`,
 #                      `--tests`, `--nologcapture`. More can be found in
 #                      https://nose.readthedocs.io/en/latest/man.html#options
+#     suite         -> Namespace for this run of tests. Required if running
+#                      under Jenkins.
```

Comment:
Can you explain more about it? It's still not clear to me what string I should give when adding new test and how it's been used from nose. Can it be an arbitrary string? Should it be unique?

Reply:
It's used to differentiate running the same tests with different arguments/python versions/etc.

Reply:
Added more explanation to the comment.